### ISSUE TITLE
Create publisher group for Chrome Web Store

### ISSUE THREAD

**Issue by dideler** [dideler/toggle-youtube-comments#26]
As a project maintainer
I want to reduce risk around the release process to the Chrome Web Store
Because urgent changes should not be blocked by the sole publisher being unavailable

Create a [publish group](https://developer.chrome.com/webstore/publish#set-up-group-publishing_1) and add trusted collaborators.

**Comment by dideler** [comment]
@1natsu172 I'm in the process of setting up the group publisher account.

Beware that Chrome Web Store requires publishers to [pay a $5 developer signup fee](https://developer.chrome.com/webstore/publish#pay-the-developer-signup-fee). This also applies to members of a Group Publisher account - [each member of the group has to pay the one-time fee](https://developer.chrome.com/webstore/publish#things-to-note) if they haven't paid the fee before. You shouldn't have to pay this fee again if you have already done so (e.g. if you've published an extension under your personal account in the past).

If you would still like to continue, I'll have to invite you to the publishing group via email. You can contact me via my email on my profile if you want to keep your email private.

**Comment by 1natsu172** [comment]
@dideler Thank you for your guide.
I sent information to your email. please confirm.

**Comment by dideler** [comment]
Created a Group Publisher account called [_Dennis & Friends_](https://chrome.google.com/webstore/developer/dashboard/uafb8833e0e606a462093ebe52d5eda3d).

It's linked to a Google Group called [_dideler-group-publisher_](https://groups.google.com/forum/#!forum/dideler-group-publisher).

Members of this group can publish any extension listed under Dennis & Friends.

Membership is invite-only. Any new members wishing to join, please leave a comment in this thread.
